pkg/virt-controller/watch/vm_test.go

```diff
@@ -43,6 +43,7 @@ import (
 
 	virtcontroller "kubevirt.io/kubevirt/pkg/controller"
 	"kubevirt.io/kubevirt/pkg/instancetype"
+	"kubevirt.io/kubevirt/pkg/pointer"
```

Comment:
duplicate import
/lgtm
/hold
Up to you if you want to proceed with address this or not